module/toggleLoginButton.js

```diff
@@ -0,0 +1,58 @@
+const emailInput = document.querySelector('.emailInput');
+const passwordInput = document.querySelector('.passwordInput');
+const loginBtn = document.querySelector('.login-button');
+
+const emailError = document.querySelector('div.email-error');
+const passwordError = document.querySelector('div.password-error');
+
+const passwordIcon = document.querySelector('.password-icon')
+
+function toggleButton() { //버튼 활성화
```

Comment:
지금도 충분히 잘 작성하셨지만, 여러 조건을 나열할 경우 변수를 이용해주시면 가독성이 좋아집니다. 
또한 toggle 메소드를 사용하면 코드를 보다 간결하게 만들 수 있어요.

```suggestion
function toggleButton() {
 const isEmailValid = !emailError.textContent;
 const isPasswordValid = !passwordError.textContent;

 loginBtn.classList.toggle('active', emailInput.value && passwordInput.value && isEmailValid && isPasswordValid);
    }
```